locales-pending/premium.ftl

```diff
@@ -238,6 +239,10 @@ dashboard-fixed-area-headline = View all exposures that are fixed or in-progress
 dashboard-exposures-area-description = We found your information exposed { $exposures_total_num } times over { $data_breach_total_num } data breaches and { $data_broker_total_num } data broker sites that are selling your personal info.
 dashboard-exposures-all-fixed-label = All fixed here!
 
+# Variables: 
+# $data_broker_total_num is the total number of data brokers selling the user’s data.
+dashboard-exposures-all-fixed-free-scan = Next <free_scan_link>start your free scan</free_scan_link> of { $data_broker_total_num } sites that may be selling your personal info.
```

Comment:
```suggestion
dashboard-exposures-all-fixed-free-scan = {
    $data_broker_total_num ->
      [one] Next <link>start your free scan</link> of { $data_broker_total_num } site that may be selling your personal info.
     *[other] Next <link>start your free scan</link> of { $data_broker_total_num } sites that may be selling your personal info.
  }
```

Please make this a plural string. Also, can we simplify the markup, using a shorter one?

Reply:
@flodolo Does this need the singular `[one]` case?
Currently the `$data_broker_total_num` will be guaranteed to be 190 (unless the vendor adds/removes more brokers).

Reply:
It can be skipped for English, but other can potentially need it if that number changes.